### ISSUE TITLE
Consider mentioning `datalad-recursiveinstall=skip`

### ISSUE THREAD

**Issue by mih** [datalad-handbook/book#327]
This is a `.gitmodule` property that can be set in order to prevent `get/install --recursive` from blindly recursing into a subdataset when no specific path argument pointing into such a dataset is given. It was originally introduced to prevent unintentional negative client-side of such a command invocation. When `installing -r`'ing from a remote URL it is impossible to know will resources will be needed (it could be hundreds of datasets, with more files than a system can handle). Dataset maintainers can use this setting to mitigate such issues for monstrous datasets. Here are all the docs on it (from `subdatasets --help`:

```
The following properties for subdatasets are recognized by DataLad
(without the 'gitmodule\_' prefix that is used in the query results):

"datalad-recursiveinstall"
    If set to 'skip', the respective subdataset is skipped when DataLad
    is recursively installing its superdataset. However, the subdataset
    remains installable when explicitly requested, and no other features
    are impaired.
```

**Comment by adswa** [comment]
That is a useful feature to know about. I put it into a footnote in b2a1cf1. Let me know what you think!

**Comment by adswa** [comment]
Done and merged with #326
